app/assets/stylesheets/components/_header.sass

```diff
@@ -1,9 +1,9 @@
 $Header-spacing-base: $Theme-spacing-base
 $Header-height-header: 100px
-$Header-background-color: $Theme-white
+$Header-color-background: $Theme-white
```

Comment:
I see what you're doing here. But wouldn't you agree, that background color should be seen as a whole world, i.e. `$Header-backgroundColor`?
What do you think?